### PR TITLE
create accordion widget for DB info

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: src/notebook_utils/notebook_utils.py [hunk 1]
```diff
@@ -71,3 +73,42 @@ def display_df(
...
+
+
+def _fetch_tables_for_namespace(spark: SparkSession, namespace: str) -> pd.DataFrame:
+    """
+    Retrieve all tables for a given namespace.
+    """
+    return spark.sql(f"SHOW TABLES IN {namespace}").toPandas()
+
+
+def _create_namespace_accordion(spark: SparkSession, namespaces: list) -> Accordion:
```
[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/notebook_utils/notebook_utils.py:92]: yea. In the next PRs, they will both be called.

File: src/notebook_utils/notebook_utils.py [hunk 2]
```diff
@@ -71,3 +73,42 @@ def display_df(
...
+        tables_df = _fetch_tables_for_namespace(spark, namespace_name)
+
+        table_content = '<br>'.join(tables_df['tableName']) if not tables_df.empty else "No tables available"
+        table_list = HTML(value=table_content)
+
+        namespace_section = VBox([table_list])
+        children_list.append(namespace_section)
+
+        accordion.children = tuple(children_list)
+        accordion.set_title(len(accordion.children) - 1, f"Namespace: {namespace_name}")
```
[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/notebook_utils/notebook_utils.py:112]: Ok, I see why I was confused. What I would do is get rid of children_list, set accordion.children to a list at the beginning of the loop, append to that directly, and then make it a tuple at the end of the loop

[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/notebook_utils/notebook_utils.py:111]: 👍

[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/notebook_utils/notebook_utils.py:111]: Why are you doing the += tuple to the list rather than just appending?

**Review by MrCreosote — APPROVED** [review]

File: src/notebook_utils/notebook_utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/notebook_utils/notebook_utils.py:109]: turns out `accordion.children` is always a tuple. So we don't the list at all.

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**